modules/charon-core/src/main/java/org/wso2/charon3/core/schema/SCIMResourceSchemaManager.java

```diff
@@ -73,6 +81,56 @@ public SCIMResourceTypeSchema getUserResourceSchema() {
         return SCIMSchemaDefinitions.SCIM_USER_SCHEMA;
     }
 
+    /*
+     * Return the SCIM User Resource Schema
+     *
+     * @return SCIMResourceTypeSchema
+     */
+    public SCIMResourceTypeSchema getUserResourceSchema(UserManager userManager)
+            throws BadRequestException, NotImplementedException, CharonException {
+
+        AttributeSchema enterpriseSchemaExtension = SCIMUserSchemaExtensionBuilder.getInstance().getExtensionSchema();
+        AttributeSchema customSchemaExtension = userManager.getCustomUserSchemaExtension();
+        if (enterpriseSchemaExtension != null) {
+            List<String> schemas = new ArrayList<>();
+            schemas.add(SCIMConstants.USER_CORE_SCHEMA_URI);
+            schemas.add(enterpriseSchemaExtension.getURI());
+            if (customSchemaExtension != null) {
+                schemas.add(customSchemaExtension.getURI());
+            } else {
+                if (log.isDebugEnabled()) {
```

Comment:
Shouldn't this be a warn log?

Reply:
fixed with https://github.com/wso2/charon/pull/323/commits/6a773e3384a5db3baab94370f9f64a904834a48d